### PR TITLE
[Linux] Retrieve the list of Android packages for installing dynamically

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
# Description
In this pull request we add logic to install android tools dynamic for linux.

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/android.sh [hunk 1]
```diff
@@ -9,6 +9,23 @@ set -e
...
+function install_android_packages {
+    minimumVersion=$( echo "$1" | sed 's/\.//g' )
+    shift
+    toolsArr=("$@")
+
+    for item in ${toolsArr[@]}
+    do
+        version=$(echo "${item##*[-;]}" | sed 's/\.//g')
+        echo "version is $version"
+        if (( $version >= $minimumVersion ))
```
Comment (al-cheb) [inline images/linux/scripts/installers/android.sh:21]: What about 1.2.1 > 1.10.10?

File: images/linux/scripts/installers/android.sh [hunk 2]
```diff
@@ -42,22 +59,20 @@ else
...
 addons=$(jq -r '.android.addon_list[]|"add-ons;" + .' $toolset)
 additional=$(jq -r '.android.additional_tools[]' $toolset)
 
 # Install the following SDKs and build tools, passing in "y" to accept licenses.
-echo "y" | ${ANDROID_SDK_ROOT}/tools/bin/sdkmanager $platforms $buildtools $extras $google_api_list $addons $additional
+echo "y" | ${ANDROID_SDK_ROOT}/tools/bin/sdkmanager $extras $google_api_list $addons $additional
 
-# Document what was added to the image
+platforms=($(${ANDROID_SDK_ROOT}/tools/bin/sdkmanager --list | sed -n '/Available Packages:/,/^$/p' | grep "platforms;android-" | cut -d"|" -f 1 | sed 's/platforms;android-//g'))
+buildTools=($(${ANDROID_SDK_ROOT}/tools/bin/sdkmanager --list | sed -n '/Available Packages:/,/^$/p' | grep "build-tools;" | cut -d"|" -f 1 | sed 's/build-tools;//g'))
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/android.sh:72]:
```
availablePlatforms=($(${ANDROID_SDK_ROOT}/tools/bin/sdkmanager --list | sed -n '/Available Packages:/,/^$/p' | grep "platforms;android-" | cut -d"|" -f 1 | sed 's/platforms;android-//g'))
availableBuildTools=($(${ANDROID_SDK_ROOT}/tools/bin/sdkmanager --list | sed -n '/Available Packages:/,/^$/p' | grep "build-tools;" | cut -d"|" -f 1 | sed 's/build-tools;//g'))
components=()
for item in ${availablePlatforms[@]}
    do
        version=$(echo "${item##*[-;]}")
        if verlte $minimumVersion $version; then
           components+=$item
        fi
    done
for item in ${availableBuildTools[@]}
    do
        version=$(echo "${item##*[-;]}")
        if verlte $minimumVersion $version; then
           components+=$item
        fi
    done
echo "y" | ${ANDROID_SDK_ROOT}/tools/bin/sdkmanager $components $extras $google_api_list $addons $additional
```

Comment (dmitry-shibanov) [reply images/linux/scripts/installers/android.sh]: Done.

File: images/linux/scripts/installers/android.sh (hunk 1 above)
Comment (dmitry-shibanov) [reply images/linux/scripts/installers/android.sh]: Done.

**Review by maxim-lobanov — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**